wapitiCore/attack/mod_log4shell.py

```diff
@@ -64,9 +66,43 @@ async def _attack_vsphere_url(self, original_request: Request):
             return
         await self._verify_header_vulnerability(malicious_request, header_target, payload, payload_unique_id, response)
 
+    async def _attack_unifi_url(self, request_url: Request):
+
+        payload_unique_id = uuid.uuid4()
+        payload = self._generate_payload(payload_unique_id)
+
+        malicious_request_username = Request(
+            path=request_url,
+            method="POST",
+            enctype="application/json",
+            post_params="{\"username\": \""+payload+"\", \"password\": "
+            "\"Letm3in_\", \"remember\": false, \"strict\": true}"
```

Comment:
There's a lot of backslashes, turning it hard to read. Let's try to use Python 3's f-strings :

`post_params=f'{"username": "{payload}", "password": "Letm3in_", "remember": false, "strict": true}'`

Reply:
Thanks 👍